### DISCUSSION TITLE
Sosyal medya takip etmekle puan gelmiyor

### DISCUSSION THREAD

**Discussion by Kriptomartinez** [IBAX-io/go-ibax#1544] [Soru Cevap - Türkçe]
Sosyal medyalara katıldım gerekeni yaptım ama hiçbir şey vermedi nedenini bilen varmı yardımcı olurmusunuz

**Comment by Yusufknk** [comment]
Bunun için yöneticilerle görüşüyoruz. İlk başta form olarak yapacaklardı ama birden fazla bot hesap ihtimali olduğu için bu yöntemi elediler. Başka bir metot üzerinde çalışıyorlar.
